materials/sql_joins.md

```diff
@@ -7,44 +7,65 @@ language: SQL
 
 ### Basic join
 
+* [`JOIN`](http://www.w3schools.com/sql/sql_join.asp) 
+    * combine rows from multiple tables
+    * based on condition
+* This query selects `species` records in the `PortalMammals_main` table and 
+  `scientificname` found in the `PortalMammals_species` table.
+    * The query links the `species` with `new_code` from `PortalMammals_species`.
+  
 ```
-SELECT yr, mo, dy, scientificname
-FROM Main
-JOIN Species ON species = new_code;
+SELECT species, scientificname
+FROM PortalMammals_main
+JOIN PortalMammsls_species WHERE species = new_code;
```

Comment:
Yeah, I waffle back and forth on that every year. The upside is as you say, the downside is that demoing the wrong way to do things tends to confuse beginners (as I now regularly notice at my daughter's martial art's class). Let's leave it as is for the moment and if I think it needs changing when I'm reviewing the material for class I'll make the changes myself.